docs/infrastructure/eks-quickstart.md

```diff
@@ -1,204 +1,159 @@
+    * **main**: Amazon EKS cluster, including the setup of all network and node infrastructure to go from zero to a fully functional cluster
+    * **setup**: Additional setup to be performed before installing Codacy on your vanilla Amazon EKS cluster
 
-## TL;DR
+## 2. Set up the Terraform state storage backend
 
-### Setting up an EKS cluster for Codacy
+The [backend](https://www.terraform.io/docs/backends/index.html) stores the current and historical state of your infrastructure.
 
-Assuming **AWS is well configured** and you fulfill all the prerequisites,
-the setup without state storage, can be done with:
+Although using the backend is optional, we recommend that you deploy it, particularly if you are planning to use these templates to make modifications to the cluster in the future:
 
-```bash
-$ export AWS_SDK_LOAD_CONFIG=1
-$ cd main/
-$ terraform init && terraform apply # should take around 15 mins
-$ cd ../setup/
-$ terraform init && terraform apply
-$ aws eks update-kubeconfig --name codacy-cluster
-```
+1. Initialize Terraform and apply the changes inside the `backend/` directory, then follow Terraform's instructions:
 
-## Deployment - Long version
+    ```bash
+    cd backend/
+    terraform init && terraform apply
+    ```
 
-### 1. `backend` - setup terraform state storage
+    An Amazon S3 bucket with a unique name to save the infrastructure state is created.
 
-The [backend](https://www.terraform.io/docs/backends/index.html) stores
-the current and historical state of your infrastructure. Deploying and using
-it is optional, but it might make your life easier if you are planning to use
-these templates to make modifications to the cluster in the future.
+1. Note the value of `state_bucket_name` in the output of the command.
 
-To deploy it run:
+1. Edit the `config.tf` files that exist in the `main/` and `setup/` directories and follow the instructions to set the name of the Amazon S3 bucket and to enable the use of the backend in those files.
 
-```bash
-terraform init && terraform apply
-```
+## 3. Create a vanilla Amazon EKS cluster
 
-inside the `backend/` folder and follow terraform's instructions.
+Create a cluster that includes all the required network and node setup:
 
-An S3 bucket with a unique name to save your state will be created. Note the value of `state_bucket_name` in the output of the command and set it on the `config.tf` file of the `main/` and `setup/` stacks where indicated.
+1. Initialize Terraform and apply the changes inside the `main/` directory, then follow Terraform's instructions:
 
-### 2. `main` - create a vanilla EKS cluster
+    ```bash
+    cd ../main/
+    terraform init && terraform apply
+    ```
 
-To create a cluster, along with all the necessary network and nodes setup
-it requires:
+    This process takes around 10 minutes.
 
-```bash
-terraform init && terraform apply
-```
+1. Consider if you want to tailor the cluster to your needs by customizing the cluster configuration.
```

Comment:
@paulopontesm we should consider if we want / need to provide more context about the variables that customers will typically want to change. Ideally, we could provide a bit of guidance in the form of "best practices".

Reply:
Or we could use those best practices as default, and assume an end user that whats to change it knows what he is doing.